iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -117,10 +117,14 @@ private async Task<MethodResponse> HandleRebootCommandAsync(MethodRequest reques
                 _logger.LogDebug($"Command: Received - Rebooting thermostat (resetting temperature reading to 0°C after {delay} seconds).");
                 await Task.Delay(delay * 1000);
 
+                _logger.LogDebug("\tRebooting...");
```

Comment:
Let me state that treating "reboot" as "reset stored telemetry data" is a stretch, and not very good for a sample IMO. That said, one sees nothing in the console about what has happened, so as long as we're living this farce, I'd at least like some console output.